modules/dnn/src/onnx/onnx_importer.cpp

```diff
@@ -1116,6 +1116,19 @@ void ONNXImporter::populateNet(Net dstNet)
                 layerParams.set("zoom_factor_y", scales.getIntValue(2));
                 layerParams.set("zoom_factor_x", scales.getIntValue(3));
             }
+            else if(node_proto.input_size() == 2 && (outShapes.find(node_proto.input(1))->second).size() == 4)
```

Comment:
What if second condition is false? Do we have working branch below?

Reply:
please add a whitespace after `if`

Reply:
we aren't using these line in the code now. may I resolve the conversation ?